gradle.properties

```diff
@@ -2,12 +2,7 @@ kotlin.code.style=official
 kotlin.mpp.enableGranularSourceSetsMetadata=true
 kotlin.native.enableDependencyPropagation=false
 
-trikot_foundation_version=2.2.1
-trikot_analytics_version=2.2.0
-trikot_streams_version=2.2.0
-trikot_viewmodels_version=2.2.1
-trikot_http_version=2.2.0
-trikot_kword_version=2.2.0
+trikot_version=3.0.0
```

Comment:
Do we really still need to specify that in the gradle.properties?

Reply:
Yep, that way we share the version between cocoapod and gradle builds.